inspirehep/modules/disambiguation/receivers.py

```diff
@@ -92,7 +92,7 @@ def _needs_beard_reprocessing(authors_before, authors_after):
         return True
 
 
-@after_record_insert.connect
+# @after_record_insert.connect
```

Comment:
Revert the merging of the disambiguation module: it's broken, it causes bugs like this.